client.go

```diff
@@ -483,7 +489,7 @@ func (c *Client) Do(req *Request) (*http.Response, error) {
 			case Logger:
 				c.RequestLogHook(v, req.Request, i)
 			case hclog.Logger:
```

Comment:
This should also be updated to the `ExtendedLogger` if the intent is to allow the use of non-stdlib and non-hclog loggers to be used.

Reply:
oops yeah, updated